pom.xml

```diff
@@ -78,8 +78,8 @@
         <fabric8.zjsonpatch.version>0.3.0</fabric8.zjsonpatch.version>
         <okhttp.version>3.12.6</okhttp.version>
         <okio.version>1.15.0</okio.version>
-        <vertx.version>3.8.5</vertx.version>
-        <vertx-juni5.version>3.8.5</vertx-juni5.version>
```

Comment:
Out of interest, was this just a typo originally or was it commonly know as `juni5` ?

Reply:
I would say just a typo.

Reply:
It is used for `vertx-junit5` - so I assume as just a typo.